bin/sl-pm.js

```diff
@@ -34,7 +34,8 @@ function printHelp($0, prn) {
 
 var argv = process.argv;
 var $0 = process.env.CMD ? process.env.CMD : path.basename(argv[1]);
-var parser = new Parser([
+var parser = new Parser(
+  [
```

Comment:
I did it this way because it produced a smaller diff and because eslint wanted me to do something I thought looked weird:
```js
var parser = new Parser([
  ':v(version)',
  'h(help)',
  'b:(base)',
  'c:(config)', // unused. left in so Upstart/systemd jobs don't crash
  'd:(driver)',
  'l:(listen)',
  'N:(no-control)', // unused. left for backwards compat.
  'P:(base-port)',
].join(''),
  argv);
```